src/date/format-properties.js

```diff
@@ -3,9 +3,10 @@ define([
 	"./pattern-re",
 	"../common/create-error/unsupported-feature",
 	"../number/symbol",
-	"../util/string/pad"
+	"../util/string/pad",
+	"../common/format-message"
```

Comment:
nitpick: sort these filepaths alphabetically please, therefore `common/format-message` should be included after the `common/..` one above.